### DISCUSSION TITLE
Multiple License Support

### DISCUSSION THREAD

**Discussion by Sherlouk** [SwiftPackageIndex/SwiftPackageIndex-Server#1763] [Ideas]
https://github.blog/changelog/2022-05-26-easily-discover-and-navigate-to-multiple-licenses-in-repositories/

I'm assuming the API will soon return an array? Any changes we need to make?

**Comment by daveverwer** [comment]
Thanks for spotting this, James.

Yes, we'll need to cope with this and decide what to do if multiple licenses are found. I'll promote this to an issue so it's tracked but I'm not sure the best way to display multiple licenses as it certainly won't fit where we have it now anymore.
